src/one_dragon_qt/widgets/multi_selection_combo_box.py

```
@@ -0,0 +1,548 @@
+       - 达到 max_width 后，开始换行
+       - 示例：min_width=200, max_width=400，宽度从 200px 逐步增加到 400px，超过后换行
+
+    3. 只有 min_width（无 max_width）：宽度无限制扩展
+       - 适用场景：基础组件，需要自由扩展
+       - 从 min_width 开始，宽度随标签数量无限增长
+       - 示例：min_width=200, max_width=None，宽度从 200px 开始无限增加
+    """
+
+    selection_changed = Signal(list)  # 当选项改变时发出信号，传递选中的值列表
+
+    def __init__(self, parent=None, max_width: int = None, fixed_width: int = None, min_width: int = None) -> None:
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)